entrypoint

```diff
@@ -74,12 +75,12 @@ case $COMPRESSION in
   gzip)
     COMPRESS="gzip"
     UNCOMPRESS="gunzip"
-    EXTENSION="gz"
+    EXTENSION="tgz"
     ;;
   bzip2)
     COMPRESS="bzip2"
     UNCOMPRESS="bzip2 -d"
-    EXTENSION="bz2"
+    EXTENSION="tbz2"
```

Comment:
@deitch may i ask why u changed this to "tbz2" ? a typo ? becasue the official extension is "bz2" 😏

Reply:
Official extension is `bz2` for bzipped files. What about tar bzipped files? I put in `tbz2` like `tgz` for tar gzipped files. Is there a different one for it?

Reply:
Actually, `tbz2` seems the accepted extension

Reply:
Merging this in. Can always update that later.

Reply:
@deitch i am a devop now for over 15 years and i have "never ever" seen the extension "tbz2" anywhere! there must be a reason for that 😂 
and [wikipedia does not mention that extension as well](https://en.wikipedia.org/wiki/Bzip2). so please, undo this to something everybody is used to know 😏

Reply:
Actually, both `tb2` and `tbz2` are commonly accepted, like `tgz` (as, of course, are `.tar.gz` and `.tar.bz2`).

Open a new issue after.

Reply:
> a need for you to change that at all

It isn't a `gz` anymore, it is a tarred gzip file, so the extension needs to indicate it. For consistency, they are all tar-gz or tar-bz2, even single file.

> if i see a "tar.gz" i know, without looking into the file, that "tar" was used. if i see ".gz" i know it´s (properly) not packed by tar.

exactly. If you saw `bz2`, you would think it is a single compressed file, not a tarred file.

Reply:
@deitch i didn´t recognize in your changes that "tar" is involved here, sorry. in that case, forget what i said. ironically most people tend to use the long version, they use "tar.gz" and "tar.bz2" - maybe it´s better readable, i don´t know, but that´s whats most commen used. would be happy if you could follow that "inofficial" standard 😏